opencga-app/app/scripts/azure/arm/azuredeploy.parameters.json

```diff
@@ -3,64 +3,67 @@
     "contentVersion": "1.0.0.0",
     "parameters": {
         "_artifactsLocation": {
-            "value": ""
+            "value": "https://raw.githubusercontent.com/opencb/opencga/azure/opencga-app/app/scripts/azure/arm/"
         },
         "_artifactsLocationSasToken": {
-            "value": ""
-        },
-        "HDInsightStorageOption": {
-            "value": "DataLake"
+            "value": "?125"
         },
         "rgPrefix": {
             "value": "opencga"
         },
         "rgLocation": {
             "value": "uksouth"
         },
-        "avereServicePrincipalAppId": {
-            "value": ""
-        },
-        "avereServicePrincipalPassword": {
-            "value": ""
-        },
-        "sshAdminKeyData": {
-            "value": ""
+        "deploymentSize": {
+            "value": 1
         },
-        "mongoDBPassword": {
-            "value": ""
+        "HDInsightStorageOption": {
+            "value": "DataLake"
         },
         "openCGAContainerImage": {
-            "value": ""
+            "value": "opencb/opencga-app:TAGHERE"
+        },
+        "batchContainerImage": {
+            "value": "opencb/opencga-batch:TAGHERE"
         },
         "ivaContainerImage": {
-            "value": ""
+            "value": "opencb/iva:TAGHERE"

```

Comment:
should we maybe default to `latest` for these docker images?

Reply:
https://hub.docker.com/r/opencb/iva/tags no latest, but is an rc tag.